sentry_sdk/integrations/aws_lambda.py

```diff
@@ -78,10 +78,10 @@ def sentry_handler(event, context, *args, **kwargs):
         with hub.push_scope() as scope:
             with capture_internal_exceptions():
                 scope.clear_breadcrumbs()
-                scope.transaction = context.function_name
                 scope.add_event_processor(
                     _make_request_event_processor(event, context, configured_time)
                 )
+                scope.set_tag("region", context.invoked_function_arn.split(":")[3])
```

Comment:
@untitaker I've added code to set "region" tag for AWS as well as for GCP integration.

Reply:
Can we prefix this with integration name such as `gcp_region`/`aws_region`. Region is too generic and it's often already hard to figure out how data made it into the system.

Reply:
okay, sure @untitaker